extension/js/common/platform/catch.ts

```diff
@@ -402,5 +402,8 @@ export class Catch {
 }
 
 Catch.RUNTIME_ENVIRONMENT = Catch.environment();
-window.onerror = Catch.onErrorInternalHandler as OnErrorEventHandler;
-window.onunhandledrejection = Catch.onUnhandledRejectionInternalHandler;
+// window is undefined in background service worker
+if (typeof window !== 'undefined') {
+  window.onerror = Catch.onErrorInternalHandler as OnErrorEventHandler;
+}
+onunhandledrejection = Catch.onUnhandledRejectionInternalHandler;
```

Comment:
maybe we can replace `window.onerror` with `onerror` like it's done for `onunhandledrejection`?

Reply:
okay